apps/jobbrowser/src/jobbrowser/views.py

```diff
@@ -114,9 +114,16 @@ def get_job(request, job_id):
 
 
 def apps(request):
+  if request.GET.get('is_mini', False):
+    return render('mini_job_browser.mako', request, {
+      'is_embeddable': request.GET.get('is_embeddable', False),
+      'is_mini': True,
```

Comment:
Can't we remove this flag now?

Reply:
Maybe. :-) I thought about it but noticed we also use it as part of the Oozie code and I was afraid it would open a can of worms and expand the scope of the change. It's noted and I'll look into in a separately to keep the change small.